docs/refs.bib

```diff
@@ -1,312 +1,320 @@
 # File for all article, arxiv and other web references
 # To add a key, Lastname_year_abbreviatedtitle/firstletteroftitle
 
-#Last name begins with A
+% Last name begins with A
 @misc{Aaronson_2018_MaxMixed,
-  author  = {Aaronson, Scott},
-  title   = "Lecture 6: Mixed States",
-  howpublished = {https://www.scottaaronson.com/qclec/6.pdf}
-  
+  author={Aaronson, Scott},
+  title="Lecture 6: Mixed States",
+  howpublished={https://www.scottaaronson.com/qclec/6.pdf},
+  url={https://www.scottaaronson.com/qclec/6.pdf}
 }
```

Comment:
If the `url` is working, you should get rid of the `howpublished` entries. Otherwise, the URL shows up twice. 

![image](https://github.com/user-attachments/assets/d32bc4e6-1dc9-4d18-8d9a-11fe69870897)

Reply:
That's a good point. I'll go ahead and remove the `howpublished` field where appropriate then. Good catch!

Reply:
Done in 65169123

Reply:
I think one of the links might have to remain as `howpublished`. 

https://github.com/vprusso/toqito/actions/runs/11193184144/job/31118220715?pr=827#step:5:1320